code/game/gamemodes/objective.dm

```diff
@@ -306,6 +306,29 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 	name = "protect nonhuman"
 	human_check = FALSE
 
+/datum/objective/jailbreak
+	name = "jailbreak"
+	martyr_compatible = TRUE //why not?
+	var/target_role_type=FALSE
+
+/datum/objective/jailbreak/find_target_by_role(role, role_type=FALSE,invert=FALSE)
+	if(!invert)
+		target_role_type = role_type
+	..()
```

Comment:
```suggestion
	target = ..()
	if (target)
		to_chat(O.target.current, "<span class='hear'>You hear something crackle in your ear for a moment before a voice speaks. \"Please stand by for a message from the Syndicate. Message as follows: <b>Hey kid, we hear you got yourself in a bit of trouble with those Nanotrasen bigwigs. We're sending an agent down to try and bust you out of that place. Just remember to let our guy do the traitorous acts (like killing) - you don't need any more blood on your hands.</b> Message ends.\"</span>")

```

Reply:
You'd have to test it, but a solution like this should work.

Reply:
oh I'm a dumbass